README.md

```diff
@@ -23,34 +23,38 @@ Get started with Hydrogen with the instructions below ⬇️
 
 **Requirements:**
 
-- `yarn` or `npm`
 - Node.js version 16.14.0 or higher
+- `npm`, `yarn` or `pnpm`
 
 **Installation:**
 
 ```bash
-# Using `yarn`
-yarn create @shopify/hydrogen
-
 # Using `npm`
-npm init @shopify/hydrogen
+npm create @shopify/hydrogen@latest
 
-# Using `npx`
-npx @shopify/create-hydrogen
+# Using `yarn`
+yarn create @shopify/hydrogen@latest
+
+# Using `pnpm`
+pnpm create @shopify/hydrogen@latest
```

Comment:
is `create` agreed?

Reply:
It's either `create` for all, or `init` for npm only and `create` for the rest. Let's go with this for now, we can always update it later.